src/App.jsx

```diff
@@ -1,33 +1,53 @@
 import './App.scss';
+import { useState } from 'react';
+
 import { MoviesList } from './components/MoviesList';
 import moviesFromServer from './api/movies.json';
 
-export const App = () => (
-  <div className="page">
-    <div className="page-content">
-      <div className="box">
-        <div className="field">
-          {/* eslint-disable-next-line jsx-a11y/label-has-associated-control */}
-          <label htmlFor="search-query" className="label">
-            Search movie
-          </label>
-
-          <div className="control">
-            <input
-              type="text"
-              id="search-query"
-              className="input"
-              placeholder="Type search word"
-            />
+const findLowQuery = (text, query) => (
+  text.toLowerCase().includes(query.toLowerCase().trim())
+);
+
+export const App = () => {
+  const [query, setQuery] = useState('');
+
+  const visibleMovies = moviesFromServer.filter(movie => (
+    findLowQuery(movie.title, query)
+    || findLowQuery(movie.description, query)
+    || null
+  ));
+
+  return (
+    <div className="page">
+      <div className="page-content">
+        <div className="box">
+          <div className="field">
+            {/* eslint-disable-next-line jsx-a11y/label-has-associated-control */}
+            <label htmlFor="search-query" className="label">
+              Search movie
+            </label>
+
+            <div className="control">
+              <input
+                value={query}
+                onChange={(event) => {
+                  setQuery(event.target.value);
+                }}
```

Comment:
it's much better to create a handler function
```suggestion
                onChange={(event) => {
                  handleInputChange(event.target.value);
                }}
```